src/Utility.java

```diff
@@ -15,9 +15,29 @@ protected static void displayMenu() {
         
     } */
 
-    /* protected ... ... handleUserMenuChoice(...) {
-        ...
-    } */
+    protected static void handleUserMenuChoice() {
+        Scanner input = new Scanner(System.in); 
+        System.out.println("Please enter selection: ");
```

Comment:
Can you make this a `print()` instead of a `println()` so the user can input their answer on the same line?